RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -678,10 +717,36 @@ public static String getUsedPluginVersion(String pluginID) {
         return LibrariesFactory.PLUGIN_VERSION_CURRENT;
     }
 
+    public static ShellCommandSet getLocalCommandSet() {
+        String localCommandSet = getApplicationProperty("localCommandSet");
+
+        // Handle default case, the local commandset ist not set.
+        if (RoddyConversionHelperMethods.isNullOrEmpty(localCommandSet)) {
+            logger.postSometimesInfo("Using BashCommandSet, localCommandSet ist not set.");
+            BashCommandSet bcs = new BashCommandSet();
+            bcs.validate();
+            return bcs;
+        }
+
+        Class<ShellCommandSet> cls = null;
+        try {
+            cls = (Class<ShellCommandSet>) LibrariesFactory.getGroovyClassLoader().loadClass(localCommandSet);
+            ShellCommandSet shellCommandSet = cls.newInstance();
+            if (shellCommandSet.validate())
+                return shellCommandSet;
+            throw new RuntimeException("The selected ShellCommandSet '${localCommandSet}' could not validate.");
```

Comment:
Do you think the user will understand this validation? Maybe add "Do you run '${localCommandSet.name}' on this system?" to give the user a hint what to change.

Also: Should this be an exception or rather directly an exit? Do we want to expose a call stack to the user?